client/cherami/client.go

```diff
@@ -58,6 +58,10 @@ const (
 	clientRetryMaxInterval                = 10 * time.Second
 	clientRetryExpirationInterval         = 1 * time.Minute
 	defaultReconfigurationPollingInterval = 10 * time.Second
+
+	destinationTag   = "destination"
```

Comment:
We maintain the tags in common/metrics/names.go so that it is in a single place and everyone can read it easily.. can you move these within that file?